<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RulerPanel</class>
 <widget class="QWidget" name="RulerPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>351</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="PushButton_Basic" name="pChiguiPanelBack">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image: url(&quot;:/images/main/graphic/fanhui@2x.png&quot;);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_chigui">
        <property name="styleSheet">
         <string notr="true">QLabel{
color: rgb(68, 68, 68);
}</string>
        </property>
        <property name="text">
         <string>尺规</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_line_9">
      <property name="styleSheet">
       <string notr="true">QLabel{
background-color: rgb(224, 224, 224);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="PushButton_2" name="pChigui2">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="PushButton_2" name="pChigui1">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="PushButton_2" name="pChigui3">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="PushButton_2" name="pChigui5">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="PushButton_2" name="pChigui4">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_21">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton_2</class>
   <extends>QPushButton</extends>
   <header>Custom/PushButton/pushbutton_2.h</header>
  </customwidget>
  <customwidget>
   <class>PushButton_Basic</class>
   <extends>QPushButton</extends>
   <header>Custom/PushButton/pushbutton_basic.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
